src/clients/mod.rs

```diff
@@ -55,19 +55,58 @@ mod test {
     }
 
     #[test]
-    fn test_endpoint_url() {
-        let spotify = ClientCredsSpotify::default();
+    fn test_api_url() {
+        let mut spotify = ClientCredsSpotify::default();
```

Comment:
Is it necessary to add the `mut` identifier to mark `spotify` mutable? It looks like the `api_url` doesn't require a mutable `Spotify`.